test/node-dht-sensor.js

```diff
@@ -87,6 +87,22 @@ describe('Read sensor', () => {
             });
         });
     });
+    describe('Asynchronously (promise)', () => {
+        it('should obtain temperature and humidity', async () => {
+            const { temperature, humidity } = await sensor.read(SENSOR_TYPE, GPIO_PIN);
```

Comment:
Shouldn't we be calling read from promises API?

Reply:
Oh, gosh. I'll fix it.